ratis-server/src/main/java/org/apache/ratis/server/impl/LeaderElection.java

```diff
@@ -186,16 +186,23 @@ public String toString() {
 
   private final RaftServerImpl server;
   private final boolean skipPreVote;
+  private final ConfAndTerm round0;
 
-  LeaderElection(RaftServerImpl server, boolean skipPreVote) {
+  LeaderElection(RaftServerImpl server, boolean force) {
```

Comment:
Argument changed to differentiate with `this.skipPreVote`.

Also to match with the caller

https://github.com/apache/ratis/blob/69263c884573b87eada10bdd269c9e9d31fb2e94/ratis-server/src/main/java/org/apache/ratis/server/impl/RoleInfo.java#L116-L121